invenio_communities/assets/semantic-ui/js/invenio_communities/community/frontpage.js

```diff
@@ -62,14 +62,16 @@ class CommunityCard extends Component {
           <Card.Header>
             {_truncate(this.props.community.metadata.title, { length: 50 })}
           </Card.Header>
-          <Card.Description>
-            <div
-              className="truncate-lines-2"
-              dangerouslySetInnerHTML={{
-                __html: this.props.community.metadata.description,
-              }}
-            />
-          </Card.Description>
+          {this.props.community.metadata.description && (
```

Comment:
minor: let try to clean out a bit the clutter and start using the props deconstruction?